components/chart/plot-bands.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Plot Bands
+page_title: Plot Bands
+description: Plot Bands in the Charts for Blazor.
+slug: chart-plot-bands
+tags: telerik,blazor,chart,plot,bands,plotbands
+published: true
+position: 45
+---
+
+# Plot Bands
+
+Plot Bands are colored ranges in the Telerik UI for Blazor Chart, the purpose of which is to highlight areas of the chart by changing its background in a predefined axis range.
+
+The plot bands appear on the chart as colored areas that fill the entire space between two grid lines on the x-axis or on the y-axis.
+
+
+## Creating Plot Bands
+
+1. Add instances of the `PlotBand` items to the `PlotBands` collection of the X or Y axes. 
+1. Set a value to the `From` and `To` parameters.
+1. Provide a valid CSS color to the `Color` parameter.
+1. (Optional) Customize the plot band further by providing values to the other exposed parameters.
+
+## Supported Chart types
+
+You can render plot bands for `Categorical` (like column, bar, area, line, candlestick) and `Numerical` (like bubble, scatterline, scatter) charts.
```

Comment:
How do we configure plot bands in the categorical X axis? For example, I can seem to configure a plot band from "Q2" to "Q3". What is supported and what is not?

Reply:
You can, quirky tho, you need to define between which two major grid lines are the Plot bands

    <ChartCategoryAxes>
        <ChartCategoryAxis Categories="@xAxisItems">
            <ChartCategoryAxisPlotBands>
                <ChartCategoryAxisPlotBand From="2" To="3" Color="red" Opacity="0.4">
                </ChartCategoryAxisPlotBand>
            </ChartCategoryAxisPlotBands>
        </ChartCategoryAxis>
    </ChartCategoryAxes>
    
 This will result in a Plot band between the third and fourth (Q3) due to the indexes.